client/my-sites/site-monitoring/metrics-tab.tsx

```diff
@@ -173,7 +182,10 @@ export const MetricsTab = () => {
 				></SiteMonitoringPieChart>
 			</div>
 			<SiteMonitoringBarChart
-				title={ __( 'Requests by HTTP Response Code' ) }
+				title={ __( 'Requests by HTTP response code' ) }
+				tooltip={ __(
+					'Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.'
```

Comment:
Number of requests categorized by their HTTP response code. Hover over each entry in the legend for detailed information.